middleware.js

```diff
@@ -1,12 +1,11 @@
 const jwt = require('jsonwebtoken');
-
 module.exports = function(req, res,next) {
     try{
         let token = req.header('x-token');
         if(!token){
             return res.status(400).send('Token Not found');
         }
-        let decode = jwt.verify(token,'jwtconfidential');
+        let decode = jwt.verify(token,process.env.JWT_SECRET);
```

Comment:
Explain this

Reply:
We are storing JWT secret key in a dotenv file and accessing it.